src/platform/apollolake/include/platform/lib/dma.h

```diff
@@ -54,6 +54,7 @@
 #define DMA_HANDSHAKE_SSP5_RX	13
```

Comment:
Lets provide more details here. Commit message should read: DMA controllers on i.MX8 provide interrupts per channel and those interrupts might fall on different IRQ steer lines. In order to compute the real interrupt number we need both DMA irq line number and DMA irq line name.